src/mscorlib/Common/PinnableBufferCache.cs

```diff
@@ -46,7 +46,7 @@ internal sealed class PinnableBufferCache
 #pragma warning disable 618
         [EnvironmentPermission(SecurityAction.Assert, Unrestricted = true)]
```

Comment:
BTW: These Permission attributes can be stripped too - they do not do anything in CoreCLR.